sql/core/src/test/resources/sql-tests/inputs/selfjoin-with-transform.sql

```diff
@@ -0,0 +1,6 @@
+create temporary view t as select * from values 0, 1, 2 as t(a);
+
+WITH temp AS (
```

Comment:
Could you move this test into `transform.sql`?

Reply:
+1

Reply:
updated